boards/x86/qemu_x86/board.cmake

```diff
@@ -9,6 +9,7 @@ endif()
 if(CONFIG_X86_64)
   set(QEMU_binary_suffix x86_64)
   set(QEMU_CPU_TYPE_${ARCH} qemu64,+x2apic)
+  list(APPEND QEMU_EXTRA_FLAGS -machine q35)
```

Comment:
I would expect to see an essential option like `-machine` explicitly specified for both x86 and x86-64 in `set(QEMU_FLAGS_${ARCH}` as do the rest of the boards; but, not terribly important.